Week3/homework/step3-bonus.js

```diff
@@ -3,8 +3,16 @@
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 function makeUnique(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  const sortedArr = arr.sort();
+  const uniqueArr = [];
+
+  for (let i = 0; i < sortedArr.length; i++) {
+    if (sortedArr[i] !== sortedArr[i + 1]) {
+      uniqueArr.push(sortedArr[i]);
+    }
+  }
+
+  return uniqueArr;
```

Comment:
Aahaan. This is a lot of code, isn't it? I would suggest reading about **Sets** in JavaScript and see if it helps in reducing the code required 🙂